packages/elogind.rb

```diff
@@ -3,11 +3,24 @@
 class Elogind < Package
   description 'Standalone systemd-logind fork'
   homepage 'https://github.com/elogind/elogind'
-  version '246.9.2'
+  version '246.9.2-4c20'
```

Comment:
Included in the latest release? https://github.com/elogind/elogind/releases/tag/v246.10

Reply:
Rebuilding...

Reply:
Done.